tests/lib/get-keys-from-ts.js

```diff
@@ -0,0 +1,43 @@
+import { diffString } from "json-diff";
```

Comment:
Can we add a file header comment describing what this file does? 

And separators between major sections?

See https://github.com/eslint/eslintrc/blob/main/lib/cascading-config-array-factory.js as an example. 

(Same for other files.)